src/sutton/physics/physics.java

```diff
@@ -0,0 +1,27 @@
+package sutton.physics;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class physics {
+    public static void main(String [ ] args) {
+
+         ArrayList<Projectile> plist = new ArrayList<Projectile>();
+
+         for (int i = 0; i <= 120; i ++){
+             plist.forEach((n) -> n.addTime(1));
+             int randomAmount = new Random().nextInt(6);
```

Comment:
Create a Random object ONCE and reuse it
Do not call `new Random()` multiple times in your codebase.